### PR TITLE
feat: Included 3.x rules for aws cis 1.3.0

### PR DESCRIPTION
**Pull request by m-pizarro**

### CODE REVIEW

File: src/aws/cis-1.3.0/.releaserc.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+branches:
```
Comment (tyler-dunkel) [inline src/aws/cis-1.3.0/.releaserc.yml:2]: any reason this file is included?

File: src/aws/cis-1.3.0/rules/aws-cis-1.3.0-3.3.ts [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+              value
+            }
+          }
+        }
+      }
+    }
+  }`,
+  resource: 'queryawsCloudtrail[*]',
+  severity: 'medium',
+  conditions: {
```
Comment (tyler-dunkel) [inline src/aws/cis-1.3.0/rules/aws-cis-1.3.0-3.3.ts:80]: I cant tell, does this check the first parts of the check description?

 `Ensure the AllUsers principal is not granted privileges to that <bucket> `

and

`Ensure the AuthenticatedUsers principal is not granted privileges to that <bucket> `

File: src/aws/cis-1.3.0/rules/aws-cis-1.3.0-3.5.ts [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+          lastStatus
+        }
+      }
+    }
+  }`,
+  resource: 'queryawsAccount[*]',
+  severity: 'medium',
+  conditions: {
+    path: '@.configurationRecorders',
+    array_any: {
```
Comment (tyler-dunkel) [inline src/aws/cis-1.3.0/rules/aws-cis-1.3.0-3.5.ts:100]: does this check its enabled for every region or do we need to do something similar that was done for guardDutyDetectors?

Comment (tyler-dunkel) [reply src/aws/cis-1.3.0/rules/aws-cis-1.3.0-3.5.ts:100]: looks to be correct from the rationale description.

File: src/aws/cis-1.3.0/rules/aws-cis-1.3.0-3.3.ts (hunk 2 above)
Comment (m-pizarro) [reply src/aws/cis-1.3.0/rules/aws-cis-1.3.0-3.3.ts:80]: Yes, you're right, I'll check it out. Thanks!

Comment (m-pizarro) [reply src/aws/cis-1.3.0/rules/aws-cis-1.3.0-3.3.ts:83]: @tyler-dunkel  done. This change depends on this PR: https://github.com/cloudgraphdev/cloudgraph-provider-aws/pull/63

**Review by tyler-dunkel — APPROVED** [review]

**Pull request merged**